packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -866,17 +852,8 @@ export default class CodeSubmode extends Component<Signature> {
         padding-top: var(--code-mode-padding-top);
         overflow: auto;
         flex: 1;
-      }
-
-      .code-mode-background {
-        position: fixed;
-        left: 0;
-        right: 0;
-        display: block;
-        width: 100%;
-        height: 100%;
-        filter: blur(15px);
-        background-size: cover;
+        background-color: var(--boxel-dark);
```

Comment:
I wonder... is it that the background is black and some items have a special background-color `#75707e` (for top bar, file inspector and module inspector headers) or is the whole background `#75707e` and the gutters have black background?